README.md

```diff
@@ -1,2 +1,62 @@
-# tg5
-TC39-TG5: Experiments in programming language standardization
+# TC39-TG5: Experiments in programming language standardization
+
+This repository contains documents, agendas, and notes for the [ECMAScript - Research Task Group (TC39-TG5)](https://ecma-international.org/task-groups/tc39-tg3/).
+
+See [Scope](#scope) for our mandate.
+
+## Agenda
+
+see [2024-03-27](meetings/2024/2024-03-27.md) and [the backlog](meetings/backlog.md) 👀
+
+## Meetings
+
+Meetings occur every Wednesday and appear on the [TC39 private calendar](https://github.com/tc39/Reflector#tc39-private-calendar).
+
+Meeting link: [join zoom meeting](https://us02web.zoom.us/j/81143085896?pwd=TUE3WGgrdEZmNFZJc0g4QzBHUWczdz09)
+
+<!-- DST below -->
```

Comment:
this was copied from an md file which had commented out time for DST / non-DST, so we should remove this comment, or add in the entire commented section. though I don't know how this meeting is following or not following some country's DST changes so maybe just remove for now?